doc/charm++/manual.rst

```diff
@@ -2701,6 +2701,74 @@ applications. RotateLB and RandCentLB are more useful for debugging
 object migrations. The compiler and runtime options are described in
 section :numref:`lbOption`.
 
+**TreeLB and its configuration**
+
+Recent changes in the LB infrastructure add support for a user-configurable
+hierarchical load balancing using TreeLB. While the legacy centralized
+strategies above are still supported, TreeLB allows performing load
+balancing at multiple levels and supports load balancing trees with different
+levels. For example, a 2-level tree consists of PEs and a root while a
+3-level tree consists of PEs, processes and a root at the top. A 4-level
+tree consists of PEs, proceeses, ProcessGroup and a root. The load balancing
+strategy to be used at each level and frequency at which to invoke LB
+at each level can be specified using command line parameters or using a json
+config file. We provide examples of these below:
+
+Creating a 2-level tree that uses GreedyRefine strategy at the root
```

Comment:
@matthiasdiener Is there any syntax we should use for a heading like here and below?

Reply:
We could maybe add another level of heading here, but it's not likely. One of the more silly aspects of RST vs. MD is the heading levels, which are far less clear in RST. In any case we should probably create a heading for `**TreeLB and its configuration**` etc. first, which would probably be better in another PR.